### PR TITLE
ADD: url support

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

**Comment by madisonmay** [comment]
Passing once apiv2 has url support

File: R/api_handler.R [hunk 1]
```diff
@@ -103,8 +103,12 @@ format_image <- function(img, size, min_axis=FALSE) {
     if (file.exists(img)) {
       img <- readPNG(img)
     } else { # is already base64
-      img <- base64decode(img)
-      img <- readPNG(img)
+      result <- try(readPNG(base64decode(img))) 
+      if (class(result) == "try-error") {
+        return(img) # likely a url
```
Comment (anneCarlson) [inline R/api_handler.R]: So, right now we just feed anything that isn't an image type we recognize to the server?

File: R/api_handler.R [hunk 2]
```diff
@@ -103,8 +103,12 @@ format_image <- function(img, size, min_axis=FALSE) {
...
+      } else {
+        img <- result;
+      }
     }
   } else {
```
Comment (anneCarlson) [inline R/api_handler.R]: Wait, are there 2 elses for this if statement?

File: R/api_handler.R (hunk 1 above)
Comment (madisonmay) [inline R/api_handler.R]: Yeah, and the server will deal with error handling.

File: R/api_handler.R (hunk 2 above)
Comment (madisonmay) [inline R/api_handler.R]: It's two separate if statements -- perhaps the indentation was weird for you?

**Pull request merged**